### PR TITLE
Upgrade kotlin

### PR DESCRIPTION
**Pull request by babedev**
Upgrade Kotlin version to 1.1.51 and also Android and Gradle.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+	google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-rc2'
```
Comment (kittinunf) [inline build.gradle]: 3.0.0 maybe?

File: build.gradle [hunk 2]
```diff
@@ -18,7 +19,7 @@ allprojects {
     ext {
         publishVersion = '0.3.0'
 
-        kotlinVersion = '1.1.4-3'
+        kotlinVersion = '1.1.51'
```
Comment (kittinunf) [inline build.gradle:22]: 👏

**Comment by kittinunf** [comment]
@babedev gently ping!

File: build.gradle (hunk 1 above)
Comment (babedev) [reply build.gradle]: I will change it right away

File: build.gradle [hunk 3]
```diff
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+	google()
```
Comment (kittinunf) [inline build.gradle]: Why is indent here off?

**Comment by kittinunf** [comment]
Thanks for your contribution!

**Pull request merged**